appinventor/misc/translate/script.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#########################################################
+###     OdeMessages.java converter to .properties     ###
+###   To add compatibility with translation centers   ###
+###          Recommended: OneSky, Transifex           ###
+### ------------------------------------------------- ###
+###     Copyright 2017 - Diego Barreiro, Makeroid     ###
+#########################################################
+
+
+lines = open('../../appengine/src/com/google/appinventor/client/OdeMessages.java', "r", encoding="utf-8").read().split("\n")
+out = open('OdeMessages.properties', 'w', encoding="utf-8")
+
+for line in lines:
+    if line.startswith("  @DefaultMessage"):
+        DefaultMessage = line.split('"')
+        def_message = DefaultMessage[1]
+    if line.startswith("  @Description"):
+        Description = line.split('"')
+        if Description[1] == "":
+            def_description = ""
+        else:
+            def_description = "# " + Description[1] + "\n"
+    if line.startswith("  String"):
+        def_string = line.replace(" ", "").replace("String", "").replace("();", "")
```

Comment:
What happens if someone includes an end of line comment? Right now it seems like this would get piped out to the file.

Reply:
What do you mean with this? Can you give an example?

Reply:
For example:

```java
String fooMethods();  // Does the foo.
```

Technically, we don't have any entries like this, but there's no reason we couldn't in the future.

Reply:
Another valid (but unlikely) event, which would probably be rejected in code review:

```java
String /*foobar*/ foobarMethods();
```